packages/react-data-grid/src/utils/columnUtils.ts

```diff
@@ -64,22 +65,45 @@ export function getColumnMetrics<R>(metrics: Metrics<R>): ColumnMetrics<R> {
   };
 }
 
+// get column width, fallback on defaultWidth when width is undefined
+function getWidth(
+  viewportWidth: number,
+  minWidth?: number | string,
+  defaultWidth?: number
+): number | void {
+  switch (typeof minWidth) {
```

Comment:
https://github.com/adazzle/react-data-grid/pull/1818/files#diff-71bddbe841a1657d0cf905e3011e1fd7R14

`minWidth` is defined as an optional number, so this case will never run unless the dev cheats the RDG like put `minWidth = '100' as any`, however that's not a case we want to cover inside RDG which is why we migrated to the `typescript`, so basically this function is not needed, and we just need to move the code to that `getSpecifiedWidth`.